src/SFA.DAS.AdminService.Web/Controllers/Apply/ApplicationController.cs

```diff
@@ -72,8 +71,11 @@ public async Task<IActionResult> ActiveSequence(Guid applicationId, BackViewMode
         public async Task<IActionResult> Sequence(Guid applicationId, int sequenceNo, BackViewModel backViewModel)
         {
             var application = await _applyApiClient.GetApplication(applicationId);
-            var organisation = await _apiClient.GetOrganisation(application.OrganisationId);
-            var latestWithdrawalDate = await _applyApiClient.GetLatestWithdrawalDateForStandard(application.OrganisationId, application.StandardCode);
+            var organisation = await _organisationsApiClient.Get(application.OrganisationId);
+
+            var latestWithdrawalDate = application.StandardCode != null
```

Comment:
Previously the API client in the admin service was interpreting 404 Not Found as de-serializing the blank body of the 404 return - which resulted in null. The API client which is now used from the Assessor service raises an exception for a 404.
A 404 is returned because passing null as the standard code cannot be matched to an endpoint - in this case we want null as the latest withdrawal date - so easiest solution is to not call the method as we know the answer is null when there is no standard code to check